ai_creative_format_service.py

```diff
@@ -162,9 +158,21 @@ def _get_default_standard_formats(self) -> List[FormatSpecification]:
                 name="In-Stream Video",
                 type="video",
                 description="Standard in-stream video ad",
-                duration_seconds=30,
-                max_file_size_kb=10240,
-                specs={"codecs": ["h264", "vp9"], "bitrate": "max 2500kbps", "formats": ["mp4", "webm"]}
+                assets=[
+                    {
+                        "asset_id": "video_file",
+                        "asset_type": "video",
+                        "required": True,
+                        "name": "Video File",
+                        "acceptable_formats": ["mp4", "webm"],
+                        "duration_seconds": 30,
+                        "max_file_size_mb": 10,
+                        "additional_specs": {
+                            "codecs": ["h264", "vp9"],
```

Comment:
shouldn't these be part of the asset definition? doesn't video asset have these?